src/frontend/hooks/server/invites.ts

```diff
@@ -0,0 +1,37 @@
+import {useMutation, useSuspenseQuery} from '@tanstack/react-query';
+import {useApi} from '../../contexts/ApiContext';
+
+export const INVITE_KEY = 'invites';
+
+export function useInvites() {
```

Comment:
nit: would make the name here more specific e.g. `usePendingInvites()`

Reply:
https://github.com/digidem/comapeo-mobile/pull/206/commits/98c4cda753be4dbb96c252b51e57f1d1bd031991